sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -7,22 +7,104 @@
 from ...fields import consts
 from ..xcvr_api import XcvrApi
 
+import logging
+from ...fields import consts
+from ..xcvr_api import XcvrApi
+from .cmisCDB import CmisCdbApi
+from .cmisVDM import CmisVdmApi
+import time
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+
 class CmisApi(XcvrApi):
     NUM_CHANNELS = 8
 
     def __init__(self, xcvr_eeprom):
         super(CmisApi, self).__init__(xcvr_eeprom)
 
     def get_model(self):
+        '''
+        This function returns the part number of the module
+        '''
         return self.xcvr_eeprom.read(consts.VENDOR_PART_NO_FIELD)
 
-    def get_serial(self):
+    def get_vendor_rev(self):
+        '''
+        This function returns the revision level for part number provided by vendor 
+        '''
+        return self.xcvr_eeprom.read(consts.VENDOR_REV_FIELD)
+
+    def get_vendor_serial(self):
```

Comment:
Changing this API name(get_serial() to get_vendor_serial()) would affect the platform test cases where get_serial() is used. e.g sonic-mgmt test scripts.

Reply:
changed back to get_serial()